Automattic/wp-calypso
# Media: Cleanup unused `canvasToBlob` util

#### Changes proposed in this Pull Request

This PR cleans up the unused `toBlob` method from the media editor, as well as `canvasToBlob` which was now unused.

#### Testing instructions

Verify that editing an image still works well (try various operations, like resizing, cropping etc.)
